main/php_ini.c

```diff
@@ -348,6 +354,31 @@ static void php_load_php_extension_cb(void *arg)
 }
 /* }}} */
 
+static void *attempt_dl_load(char *path, char **errp)
```

Comment:
That's fine, ext/standard cannot be disabled. By that -  such a usage is legit, and in fact, there are quite a few of other places doing it in main.